presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -423,6 +423,55 @@ If your workload experiences the error *Timeout waiting for connection from
 pool*, increase the value of both ``hive.s3select-pushdown.max-connections`` and
 the maximum connections configuration for the file system you are using.
 
+Alluxio Configuration
+---------------------
+
+`Alluxio <https://www.alluxio.io/?utm_source=prestodb&utm_medium=prestodocs>`_
```

Comment:
let's get rid of this paragraph, since the documentation should focus on Presto.  People can follow the link to learn more about Alluxio.

Reply:
Done.